### DISCUSSION TITLE
请问安装了 VLOOK 是逆的吗？

### DISCUSSION THREAD

**Discussion by Evolution-detector** [MadMaxChow/VLOOK#95] [Q&A] [answered]
作者你好，我在日常工作中有使用 markdown 记录文字的习惯，最近希望将自己的一些文章发布到个人网站上。了解了 Hexo、Jekyll 等，觉得过于复杂；mkdocs 又没有首页封面；docsify 又需要了解大量 HTML 的语法。我没有编程经验，希望把更多精力花在内容的编辑上，而不是编程。最终找到了 VLOOK，感觉挺不错的，刚好我平时用的的也是 Typora。
我现在有以下疑问：
1. 文档中没有描述具体使用过程，这么多丰富的格式是否会大大扩展  markdown 语法，有不小的学习成本？
2. 如果我安装 VLOOK 不适应，想要用回原来简洁的 Typora 是否有便捷的方式，还是说只能重装 Typora？
3. 对 Typora 的版本有没有要求（我现在用的还是 2021 年免费时期的版本）？

**Comment by MadMaxChow** [comment, marked as answer]
抱歉现在才回复你的这个问题：
1. 有详细的在线指引文档，详见：[简介](https://madmaxchow.github.io/VLOOK/index.html)、[参考手册 Part I](https://madmaxchow.github.io/VLOOK/guide.html)、[参考手册 Part II](https://madmaxchow.github.io/VLOOK/guide2.html)

2. 不涉及重装 Typora 的问题，只需要选择非 VLOOK 的主题（如 GitHub），然后将 md 文件中不需要的扩展内容删除即可（量大的情况下可以用 UltraEdit 通过从文件中替换的方式进行批量替换）

3. 原则上建议使用 Typora 的最新版本，因为其渲染的结构会变化，VLOOK 一般都会对最新正式版做适配，但不会对旧版进行兼容。目前正式的收费版本是3设备共享，两位数的价格，值得入，哈哈哈哈～
